node/src/manager/commands/info.rs

```diff
@@ -5,7 +5,7 @@ use graph_store_postgres::{connection_pool::ConnectionPool, Store};
 
 use crate::manager::deployment::{Deployment, DeploymentSearch};
 
-fn find(
+pub fn find(
```

Comment:
Might be better to make this a method on `DeploymentSearch` if it's used in more than one place

Reply:
Fixed in b2adf2922